src/packaging/webots_distro.c

```diff
@@ -981,10 +981,6 @@ static void create_file(const char *name, int m) {
               "ValueName: \"SupportedTypes\"; ValueData: \".wbt\"; Flags: uninsdeletekey\n"
               "Root: HKA; SubKey: \"Software\\Classes\\Applications\\webotsw.exe\"; ValueType: string; "
               "ValueName: \"FriendlyAppName\"; ValueData: \"Webots\"; Flags: uninsdeletekey\n"
-              "Root: HKA; SubKey: \"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\webotsw.exe\"; ValueType: string; "
-              "ValueData: \"{app}\\msys64\\mingw64\\bin\\webotsw.exe\"; Flags: uninsdeletekey\n"
-              "Root: HKA; SubKey: \"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\App Paths\\webotsw.exe\"; ValueType: string; "
-              "ValueName: \"Path\"; ValueData: \"{app}\\msys64\\mingw64\\bin;{app}\\msys64\\usr\\bin\"; Flags: uninsdeletekey\n"
```

Comment:
This is because these paths are already added by the Windows launcher (webotsw.exe or webots.exe). So they were duplicate in the Webots PATH environment variable.

Reply:
Thank you for the explanation.